src/backend/pipeline/ttl_vacuum.c

```diff
@@ -200,15 +210,20 @@ NumSWExpiredTuples(Oid relid)
 	resetStringInfo(&sql);
 
 	stmt = (SelectStmt *) linitial(parsetree_list);
-	stmt->whereClause = get_sw_vacuum_expr(cvname);
 
+	if (IsSWContView(cvname))
```

Comment:
We shouldn't need this as well. We should just copy the SW interval to the `ttl` column of the entry in `pipeline_query`.